src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -237,35 +203,9 @@ const DateForm: React.FC<{
 
       return format(dateValue, getDateFormat(market)!, { locale: dateLocale! })
     }
-
-    const firstExternalInsurance =
-      externalInsuranceData?.externalInsuranceProvider?.dataCollection[0]
-    const renewalDate = firstExternalInsurance?.renewalDate
-
-    if (renewalDate) {
-      const externalInsuranceProvider = externalInsuranceProviders.find(
-        (provider: { externalCollectionId?: string }) =>
-          provider.externalCollectionId ===
-          firstExternalInsurance?.insuranceProvider?.toUpperCase(),
-      )
-
-      return (
-        <DataCollectedStartDateWrapper>
-          <DataCollectedStartDateValue>
-            {renewalDate}
-          </DataCollectedStartDateValue>
-          <DataCollectedStartDateDescription>
-            {textKeys.START_DATE_EXTERNAL_PROVIDER_SWITCH({
-              insuranceProvider: externalInsuranceProvider?.name,
-            })}
-          </DataCollectedStartDateDescription>
-        </DataCollectedStartDateWrapper>
-      )
-    }
-
-    return textKeys.SIDEBAR_STARTDATE_CELL_VALUE_SWITCHER()
   }
 
+  // calendar, should be broken out to a regular component
```

Comment:
Maybe add a `TODO:` here as well?

Reply:
Yes! 👌  Or just remove the comment maybe. Think I left it by accident.

Reply:
Realised why I left it there, so remade it into a `TODO`.